common/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskResult.java

```diff
@@ -68,6 +68,9 @@ public enum Status {
     @Hidden
     private Any outputMessage;
 
+    @ProtoField(id = 9)
+    private boolean indexToEs;
```

Comment:
indexToEs is a boolean type. Should we name it as "isIndexedToEs" so that variable name can sound more like a boolean.

Reply:
the getmethod for boolean variable will have is (public boolean isIndexToEs()) and the prefix. normally boolean variable names does not have is prefix